08-cloudwatch-logs/README.md

```diff
@@ -207,6 +226,9 @@ _What type of events might be important to track in an AWS account? If
 you were automating mitigating actions for the events, what might they
 be and what AWS resource(s) would you use?_
 
+> Creation, deletion, and modification of resources. We could use cloudwatch to trigger sns notifications of things like excessive resource creation.
+
```

Comment:
Events to watch might include:

- Account issues like IAM user creations, role creations, policy attachments, MFA requirement changes or password requirement changes
  - Assumption of a role on behalf of any IAM admin users
  - Addition of IAM users to admin groups/granting admin privileges
- Launching expensive resources like EC2 GPU instance types
- Major network changes like VPC peering, NACL group changes, or security group changes

Mitigating actions might include:

- Disabling ec2.LaunchInstance after suspicious instance types have been launched
- Resetting resources by updating CloudFormation stacks. This is much easier to do, and to trust, when all your infrastructure comes from code.